restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Call.java

```diff
@@ -189,10 +192,11 @@
     private SipURI from;
     private javax.servlet.sip.URI to;
     // custom headers for SIP Out https://bitbucket.org/telestax/telscale-restcomm/issue/132/implement-twilio-sip-out
-    private Map<String, String> headers;
+    private Map<String, String> rcmlHeaders;
+    private Map<String, ArrayList<String>> headers;
```

Comment:
have added the comments and renamed `headers` to `extensionHeaders`
When i think about it `extensionHeaders` is used for too many things, potentially shouldve separated the header property modification and header addition into separate maps.